gramex/handlers/mlhandler.py

```diff
@@ -112,14 +114,23 @@ def setup(cls, data=None, model={}, config_dir='', **kwargs):
             data = cls._filtercols(data)
             data = cls._filterrows(data)
             cls.model = cls._assemble_pipeline(data, mclass=mclass, params=params)
-
             # train the model
             target = data[target_col]
             train = data[[c for c in data if c != target_col]]
+            # cross validation
+            cls.cross_validation(train,target)
             gramex.service.threadpool.submit(
                 _fit, cls.model, train, target, cls.model_path, cls.name)
         cls.config_store.flush()
-
+  
+    @classmethod
+    def cross_validation(cls,train,target):
+        cv = cls.get_opt('cv',True)
+        if cv:
+            CVscore = cross_val_score(cls.model.steps[-1][1], X=train, y=target, cv=cv)
+            CVavg = sum(CVscore)/len(CVscore)
```

Comment:
If `cv` is `True`, or unspecified, here it will pass to `cross_val_score` as `cv=True` - which is not a valid input.